share/hooks/ft/host_error.rb

```diff
@@ -283,6 +290,15 @@ def states_xpath(*arr)
             log "delete #{vm_id}"
             vm.delete
         when :migrate
+            vm_ds_id  = vm.retrieve_elements("/VM/HISTORY_RECORDS/HISTORY[position()=last()]/DS_ID")[0]
```

Comment:
linting " --> ' I think there is no interpolation so single quotes?

Reply:
Also is this "exception" save? I mean retrieve elements what happen if no vm_sd_id is found?

Reply:
good catch (pun intended) :p

Reply:
It should be safe, as in the vm_id array should only contain VMs which were either in states ACTIVE, SUSPENDED, or POWEROFF (3, 5, 8) when pulled and should therefore have been scheduled at least once.   I'll add a check for it in the event some edge case occurs that I'm unaware of.